.env

```diff
@@ -48,7 +48,7 @@ riesgosWpsGeoserverPassword=geoserver
 # of the docker compose.
 # We also have the same for port & protocol of the resulting url.
 riesgosWpsAccessServerHost=localhost
-riesgosWpsAccessServerPort=8082
+riesgosWpsAccessServerPort=80
```

Comment:
Fine by me - that port's still free - also with DLR's docker-compose taken into consideration.